datagrepper/templates/index.html

```diff
@@ -44,7 +44,7 @@
   <body>
 
                 <div class="masthead">
-                  <div class="container-narrow"><img src="{{datagrepper_logo}}" height=40px /></div>
+                  <div class="container-narrow"><img src="{{datagrepper_logo}}" height="40px"></div>
```

Comment:
Are img tags supposed to be closed with a "/" ?

Reply:
According to https://www.w3.org/TR/2012/WD-html-markup-20120329/syntax.html#void-element `img` is a `void element` which makes the closing `/` optional. I can add it if you prefer.

Reply:
I guess my info were outdated, thanks :)